skimage/feature/tests/test_hog.py

```diff
@@ -51,6 +51,20 @@ def test_hog_image_size_cell_size_mismatch():
     assert len(fd) == 9 * (150 // 8) * (200 // 8)
 
 
+def test_hog_odd_cell_size():
+    img = np.zeros((3, 3))
+    img[2, 2] = 1
+
+    correct_output = np.zeros((9, ))
+    correct_output[0] = 0.4999775
```

Comment:
would rather set the value to `0.5` as it may be more intuitive and later in the test set `decimal` precision to `1` which would become also more robust for test on different OS
https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.testing.assert_almost_equal.html

Reply:
Updated to use `decimal=1`.
It got better, thank you!